articles/101_build_tool.md

```diff
@@ -233,7 +233,7 @@ It is up to the implementation to decide if it only supports the standard CMake
 
 In ROS 2 the concept of the *devel space* has intentionally been removed.
 In the future it might be feasible to provide the concept of *symlinked installs* in ROS 1 to provide a similar benefit without the downsides.
-Therefore not supporting this concept in the universal build tool is considered a viable path forward.
+However, this design document will assume that the *devel space* will remain in ROS 1.
```

Comment:
I don't this this article should that claim either. Maybe it would be better to just state the fact that ROS 1 currently supports the devel space and the universal build tool doesn't. But exclude any future decision and defer that to be dealt with on the ROS 1 side (maybe in a REP).

Reply:
It wasn't my intention to claim either. I was just taking the "status quo" as the most likely future. I think that's the best assumption to take given the circumstances. Note an assumption is not an assertion or claim. Just what we have to assume until it changes.

I'm not sure how I would change this. Would you like me to remove it, or modify it some specific way?

Reply:
I read the assumption slightly different. Like "we have thought about it and for now think it will stay that way". I think it should more like "at this point we are not making a decision how to move forward with this in ROS 1".

Maybe:
> The direction forward in ROS 1 - if the new build tool needs to support the devel space or if the concept of the devel space is deprecated at some point (maybe even implicitly if the new build tool doesn't support it) - has not been decided yet and is out of scope of this article.

Reply:
> Like "we have thought about it and for now think it will stay that way". I think it should more like "at this point we are not making a decision how to move forward with this in ROS 1".

Those are the same thing... basically "at this point we are not making a decision how to move forward with this in ROS 1, therefore it will stay that way until we do make a decision".

I'm just removing the sentence. The previous lines describe the situation sufficiently.